package.json

```diff
@@ -27,6 +27,7 @@
     "ember-cli-app-version": "^1.0.0",
     "ember-cli-content-security-policy": "0.4.0",
     "ember-cli-dependency-checker": "^1.1.0",
+    "ember-cli-es5-shim": "0.1.1",
```

Comment:
It's for PhantomJS. This polyfill is in `devDependencies` and will not make it into app builds.